src/components/AboutUsIntroduction/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import useTranslation from 'next-translate/useTranslation'
+import Trans from 'next-translate/Trans';
```

Comment:
Please remove this semicolon.

```suggestion
import Trans from 'next-translate/Trans'
```